client/src/pages/TurnsHistory/TurnTable.js

```diff
@@ -10,7 +10,9 @@ import {
   Title,
 } from './styled'
 
-const TurnsTable = ({ turnHistory, turnUser }) => {
```

Comment:
I think it makes more sense to keep this as `TurnsTable`. If you change it, remember to change the file name too please